var/spack/repos/builtin/packages/bml/package.py

```diff
@@ -38,3 +38,6 @@ class Bml(CMakePackage):
 
     depends_on("blas")
     depends_on("lapack")
+
+    def cmake_args(self):
+        return ['-DBUILD_SHARED_LIBS=ON']
```

Comment:
Would it make sense to control this with a variant like in https://github.com/LLNL/spack/blob/develop/var/spack/repos/builtin/packages/ldc/package.py ? I think this will exclusively build the shared libs, vs. both shared and static.

Reply:
cmake can only build static or shared, not both at the same time.

In an earlier review, I was told shared is prefered, I couldn't find the discussion anymore, but maybe @adamjstewart or @alalazo remember.

Reply:
The original author @rspavel may have had use for the static libs (which I think is the default when `BUILD_SHARED_LIBS` is not specified). I'll say it would be easy to think about if this was added as an option.

> In an earlier review, I was told shared is prefered

Was this one of your PRs? In any case I think that may argue for making the default `shared` but it doesn't sound wrong to optionally build static.